### ISSUE TITLE
As of April 10, 2017, met.refeds.org data appears to be stale

### ISSUE THREAD

**Issue by canariecaf** [GEANT/met#23]
I am working on entities that I can see in other federation aggregates but met is not reporting them.
I am specifically looking at:
https://met.refeds.org/met/entity/https%253A%252F%252Flogin.queensu.ca%252Fidp%252Fshibboleth/

and know that as of April 10, 2017, 9am UTC-4 that this specific entity has been published at least over the weekend.

It may be useful to have the last time that federations aggregate was refreshed in Met on the user interface.

Thanks.

**Comment by biancini** [comment]
Hi, sorry for the late answer.
I'm trying to understand the problem but I can't get what it is.
The entity https://login.queensu.ca/idp/shibboleth is present in MET and is marked as part of the CAF federation:
https://met.refeds.org/met/entity/https%253A%252F%252Flogin.queensu.ca%252Fidp%252Fshibboleth/?federation=caf-federation

In the federation page: https://met.refeds.org/met/federation/caf-federation/, side by side with the pie chart it is possible to read the last update time for federation metadata: "Metadata updated on: April 15, 2017" (as per when I'm writing).

Was the entity missing in MET when you wrote? Or what is the problem you're experiencing?
Just to check, MET retrieves CAF metadata from this URL: https://caf-shib2ops.ca/CoreServices/caf_metadata_signed_sha256.xml.

Thanks

**Comment by canariecaf** [comment]
Thanks for looking into this. It seems that the MET tool was lagging significantly and now seems more accurate in the case of this host.    I did another search just now (many days later) and the entity seems to have been detected by MET in the locations I was expecting. 
The lag time is the key and understanding how many hours old the last aggregate and next refresh of a given one is an area that would help shape expections on when I would see an entity show up. (ie if I published this on Time=0, then Time+some delay that I could forecast=when it would show up elsewhere)
